content/guides/server-sent-events.adoc

```diff
@@ -10,26 +10,164 @@ toc::[]
 
 _by {author}_, _{revdate}_
 
-== Placeholder
-
-**Pull requests welcome!**
-
 == Welcome
 
+Server-sent events make it easy to stream events to a client application. They are appropriate when you only need a unidirection flow of events from the server to the client (if you need a bidirectional connection then Websockets are a better choice).
+
 == What You Will Learn
 
+After reading this guide, you will be able to:
+
+- Configure Pedestal to accept server-sent event connections.
+- Send events on the connection using core.async.
+- Connect to the event stream with a JavaScript client.
+
 == Guide Assumptions
 
+This guide is for intermediate users who have worked through some
+sample applications or the first few link:hello-world[Hello World]
+guides. In particular, you should know how to run a build, start a
+REPL, and start and stop your server.
+
+You do not need to know any other Clojure web frameworks, but if you
+do, you may find some of the comparisons useful. If not, that's OK,
+just skip those sections.
+
 == Getting Help if You're Stuck
 
+If you get stuck at any point in this guide, please submit an
+https://github.com/pedestal/docs/issues[issue] about this guide or hop
+over to the
+https://groups.google.com/forum/#!forum/pedestal-users[mailing list]
+and raise your hand there. You can often find help in the "pedestal"
+channel of the https://clojurians.slack.com[Clojurians Slack team.]
+
 == Where We Are Going
 
+We'll be creating a new Pedestal service project, adding a simple route to stream a counter using server-sent events, and add a simple HTML page that uses a JavaScript EventSource to open a connection to the stream.
+
 == Before We Begin
 
-== Content Section 1
+This guide will use http://leiningen.org/[Leiningen] to generate and
+build a template project. If you haven't already installed it, please
+take a few minutes to set it up.
+
+== Generate Our New Project
+
+----
+$ lein new pedestal-service myapp
+Generating a pedestal-service application called server-sent-events.
+$ cd server-sent-events
+----
+
+== Add Hiccup
+
+We will be using Hiccup to generate our index page, so we will need to add the dependency in our project.clj.
+
+[source,clojure]
+.project.clj
+----
+include::server-sent-events/project.clj[tags=add-hiccup]
+----
+
+== Clean Up
+
+Here's the service.clj file generated for us:
+
+[source,clojure]
+.src/server_sent_events/service.clj
+----
+include::server-sent-events/src/server_sent_events/service.clj[tags=original-service]
+----
+
+Let's simply things by uncommenting the 'Terse/Vector-based routes' section and removing uneccessary code.
```

Comment:
This line says "uncommenting" the terse/vector-based routes, but the code looks more like you commented them out.